core/server/src/main/java/alluxio/worker/block/BlockMetricsReporter.java

```diff
@@ -11,36 +11,39 @@
 
 package alluxio.worker.block;
 
-import alluxio.Constants;
 import alluxio.StorageTierAssoc;
 import alluxio.WorkerStorageTierAssoc;
-import alluxio.worker.WorkerSource;
+import alluxio.metrics.MetricsSystem;
 
-import java.util.List;
+import com.codahale.metrics.Counter;
 
-import javax.annotation.concurrent.NotThreadSafe;
+import javax.annotation.concurrent.ThreadSafe;
 
 /**
  * This class listens on block events and increases the metrics counters.
  */
-@NotThreadSafe
+@ThreadSafe
 public final class BlockMetricsReporter extends AbstractBlockStoreEventListener {
-  private final WorkerSource mWorkerSource;
   private final StorageTierAssoc mStorageTierAssoc;
 
+  public static final String BLOCKS_ACCESSED_NAME = "BlocksAccessed";
+
+  private static final Counter BLOCKS_ACCESSED = MetricsSystem.workerCounter(BLOCKS_ACCESSED_NAME);
```

Comment:
I see a different pattern here: instead of having a `private static Metrics` class, we refer the counter directly. I prefer to have a consistent pattern across our codebase.